{
  "name": "collect",
  "version": "1.0.0",
  "description": "Collect has a variety of long-running tasks which require time and resources on the servers. As it stands now, once we have triggered off a long-running task, there is no way to tap into it and pause/stop/terminate the task, upon realizing that an erroneous request went through from one of the clients (mostly web or pipeline).",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/aarshic/Collect.git"
  },
  "author": "",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/aarshic/Collect/issues"
  },
  "homepage": "https://github.com/aarshic/Collect#readme",
  "dependencies": {
    "async": "^2.6.2",
    "cluster": "^0.7.7",
    "csv-headers": "^1.0.0",
    "csv-parse": "^4.3.3",
    "express": "^4.16.4",
    "json": "^9.0.6",
    "leftpad": "0.0.1",
    "mysql": "^2.16.0",
    "nodemon": "^1.18.10",
    "request": "^2.88.0",
    "worker-queue": "^0.1.1"
  }
}
